### DISCUSSION TITLE
Problem with annotating images using PPOCRLabel tool

### DISCUSSION THREAD

**Discussion by aliiafkari** [PaddlePaddle/PaddleOCR#13878] [Q&A]
Dear All,
I am encountering an issue with annotating images using the PPOCRLabel tool for text detection and the E2E task. Specifically, I am unable to obtain the expected annotation format as outlined in the text detection guidelines.

The current output generated by PPOCRLabel for an image, for example, is as follows:
```
train/img_0.jpg	[{"transcription": "پار", "points": [[331, 493], [331, 493], [331, 493], [331, 493]], "difficult": false}, 
{"transcription": "###", "points": [[331, 493], [331, 493], [331, 493], [331, 493]], "difficult": false}, 
{"transcription": "###", "points": [[331, 493], [331, 493], [331, 493], [331, 493]], "difficult": false}, 
{"transcription": "لطفا", "points": [[331, 493], [331, 493], [331, 493], [331, 493]], "difficult": false}, 
{"transcription": "پارک", "points": [[331, 493], [331, 493], [331, 493], [331, 493]], "difficult": false}, 
{"transcription": "نکنید", "points": [[331, 493], [331, 493], [331, 493], [331, 493]], "difficult": false}]
```
However, according to the documentation provided by PaddleOCR, the expected format should be:
```
"Image file name              Image annotation information encoded by json.dumps"
ch4_test_images/img_61.jpg    [{"transcription": "MASA", "points": [[310, 104], [416, 141], [418, 216], [312, 179]]}, {...}]
```

Could anyone kindly assist in resolving this issue to achieve the correct annotation format?

Thank you in advance for your help.

**Comment by VishyAnand28** [comment]
Hi,

Please have a look at the text file of those annotated format, for instance, annotated file for ICDAR 2015 dataset is available here: https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/dataset/ocr_datasets_en.md

Important lines:
1. The annotation file formats supported by the PaddleOCR text detection algorithm are as follows, separated by "\t"
2. The points in the dictionary represent the coordinates (x, y) of the four points of the text box, arranged clockwise from the point at the upper left corner.
3. Transcription represents the text of the current text box. When its content is "###" it means that the text box is invalid and will be skipped during training.

Follow these and you can get it done.
